### PR TITLE
V1 fixes

### PR DESCRIPTION
**Pull request by fabriziovigevani**
Applies fixes done in v1.

### CODE REVIEW

File: src/components/BalanceModule.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ function BalanceModule() {
                 color: ${theme.contentSecondary};
               `}
             >
-              ({inactivePct}% idle)
+              ({parseFloat(inactivePct).toFixed(2)}% idle)
```
Comment (rperez89) [inline src/components/BalanceModule.js:70]: is this a js native function?

File: src/components/BalanceToken.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ const BalanceToken = ({ amount, symbol, color, size, icon }) => {
           icon || tokenIconUrl(tokenAddress, symbol, network && network.type)
         }
       />
-      {amount !== undefined ? splitAmount(amount.toFixed(3)) : ' - '}
+      {amount !== undefined ? splitAmount(amount, decimals) : ' - '}
```
Comment (rperez89) [inline src/components/BalanceToken.js:42]: should we use null instead of undefined?

File: src/hooks/useFilterProposals.js [hunk 3]
```diff
@@ -18,11 +18,11 @@ import { checkInitialLetters } from '../lib/search-utils'
 const NULL_FILTER_STATE = -1
 const STAKE_STATUS_FILTER_SUPPORTED = 1
 const STAKE_STATUS_FILTER_NOT_SUPPORTED = 2
-const EXECUTION_STATUS_FILTER_OPEN = 0
-const EXECUTION_STATUS_FILTER_ACCEPTED = 1
-const EXECUTION_STATUS_FILTER_CANCELLED = 2
-const TYPE_FILTER_FUNDING = 0
-const TYPE_FILTER_SIGNALING = 1
+const EXECUTION_STATUS_FILTER_OPEN = 1
```
Comment (rperez89) [inline src/hooks/useFilterProposals.js:21]: maybe we can alphabetize those constants

**Review by rperez89 — COMMENTED** [review]
LGTM some minor comments!

File: src/components/BalanceModule.js (hunk 1 above)
Comment (fabriziovigevani) [reply src/components/BalanceModule.js:70]: Yes, its a float property function

File: src/hooks/useFilterProposals.js (hunk 3 above)
Comment (fabriziovigevani) [reply src/hooks/useFilterProposals.js:21]: In this case i think its preferable the order to be by type of filter (given by the number) wdyt?

File: src/components/BalanceToken.js (hunk 2 above)
Comment (fabriziovigevani) [reply src/components/BalanceToken.js:42]: We could check for both values but i wouldn't drop the `undefined` check

**Review by rperez89 — APPROVED** [review]
LGTM

**Pull request merged**